README.md

```diff
@@ -11,6 +11,12 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
 
 ## List of all plugins! 🧪☕️⚡️
 
+* [NameValidator](https://github.com/Rozen4334/NameValidator/releases/tag/v1.3) by Rozen4334
+  * Disallows inappropriate names, nonalphanumerical, +23 character names & an configurable amount of spaces in a name. 
+  * Most of this is configurable in the config file, filtered statements can be added and removed ingame.
+  * Tested on TShock 4.5.4
+  * [Download](https://files.catbox.moe/zwjwlh.zip)
```

Comment:
Extensions matches nuget release 3.2.500 @ `94b875289f066c27414b53cb6834fcbac553eca1`
```
~/Downloads via 💎 v3.0.1
❯ shasum /Users/particles/Downloads/lib/Extensions.dll
94b875289f066c27414b53cb6834fcbac553eca1  /Users/particles/Downloads/lib/Extensions.dll

~/Downloads via 💎 v3.0.1
❯ shasum /Users/particles/Downloads/zwjwlh/Extensions.dll
94b875289f066c27414b53cb6834fcbac553eca1  /Users/particles/Downloads/zwjwlh/Extensions.dll
```

Reply:
Remaining libraries in zwjwlh ok